IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -85,6 +86,91 @@ - (void)setCallback:(MSIDLogCallback)callback
     });
 }
 
+- (MSIDLogLevel)level
+{
+    if (self.loggerConnector) return self.loggerConnector.level;
+    
+    return _level;
+}
+
+- (BOOL)PiiLoggingEnabled
+{
+    BOOL result;
+    @synchronized (self)
+    {
+        if (self.loggerConnector)
+        {
+            result = self.loggerConnector.PiiLoggingEnabled;
+        }
+        else
+        {
+            result = _PiiLoggingEnabled;
+            
+        }
+    }
+    return result;
+}
+
+- (void)setPiiLoggingEnabled:(BOOL)PiiLoggingEnabled
+{
+    @synchronized (self)
+    {
+        _PiiLoggingEnabled = PiiLoggingEnabled;
+    }
+}
+
+- (BOOL)NSLoggingEnabled
```

Comment:
nit: should we follow naming conventions and have properties start with lowercase?

Reply:
I kept the current style, let's discuss offline.